extensions/wikia/GlobalNavigation/styles/GlobalNavigationUserLoginDropdown.scss

```diff
@@ -1,28 +1,28 @@
 @import 'skins/shared/mixins/box-shadow';
 @import 'skins/shared/mixins/gradient';
 @import 'skins/shared/mixins/placeholder';
-@import 'extensions/wikia/Venus/styles/variables';
-@import 'extensions/wikia/Venus/styles/mixins';
-@import 'extensions/wikia/GlobalNavigation/styles/GlobalNavigationColors';
-@import 'extensions/wikia/GlobalNavigation/styles/GlobalNavigationVariables';
+@import 'GlobalNavigationColors';
+@import 'GlobalNavigationVariables';
 
 .AccountNavigation.active .UserLoginDropdown {
 	display: block;
 }
 
+$user-login-content-padding: 19px;
+
 .UserLoginDropdown {
```

Comment:
True, but it would require some additional changes to (i.e. tracking) and also would require extensive QA. I've decided not to do that this time around - we can create seaparate ticket for that.